libraries/standard/http/utest/http_send_utest.c

```diff
@@ -1083,8 +1083,8 @@ void test_HTTPClient_Send_null_response( void )
 {
     HTTPStatus_t returnStatus = HTTP_SUCCESS;
 
-    memcpy( requestHeaders.pBuffer, 
-            HTTP_TEST_REQUEST_PUT_HEADERS, 
+    memcpy( requestHeaders.pBuffer,
+            HTTP_TEST_REQUEST_PUT_HEADERS,

```

Comment:
Any changes from above are from uncrustify (using the VS Code plugin)

Reply:
Thanks